### PR TITLE
Fix travis & microphone example

### PR DESCRIPTION
**Pull request by d3m3vilurr**
1. travis broken after #18 (but looks unactivated this repo)
2. microphone broken after #38

### CODE REVIEW

File: microphone/src/main.c [hunk 1]
```diff
@@ -110,21 +117,21 @@ int main(int argc, char *argv[]){
 		printf("\nYour microphone is %s\n\n", sceAudioInGetStatus(1)?"disabled.": "enabled. ");
 		printf("\nSimple VU meter: (sensitivity = %3d)\n\n\n", sensitivity);		
 		for (i = 0; i < MAX_VU; i++){
-			psvDebugScreenCoordX = 56;
+			coordX = 56;
 			if (i < average){
 				if (MAX_VU/2 > i)
-					psvDebugScreenSetBgColor(COLOR_GREEN);
+					psvDebugScreenSetBgColor(0xFF00FF00);
```
Comment (devnoname120) [inline microphone/src/main.c]: Why?

Comment (d3m3vilurr) [reply microphone/src/main.c]: I like that, but @yne removed in #38 and explained [reason](https://github.com/vitasdk/samples/pull/38#discussion_r153209221)

maybe really need this macro, we should revert psvDebugScreenSet(Bg/Fg) and color macros in another pr :)

Comment (pyroesp) [reply microphone/src/main.c]: Maybe the colors should be changed to the CSI sequence thing ?
printf("\e[....m");

Comment (d3m3vilurr) [reply microphone/src/main.c]: yes, we can. just easy/non-easy problem :>

**Pull request merged**